work/generate_conversion_report.py

```diff
@@ -168,13 +201,22 @@ def html(site_folder, output_file, output_url, config, SITE_ID):
             site_title_tag.append(sitelink)
 
             # Sort the issues list
-            found_items = list(filter(lambda i: i['is_found'], config['issues']))
+            found_items = list(filter(lambda i: i['is_found'] or isinstance(i['is_found'], dict), config['issues']))
             sorted_items = sorted(found_items, key=lambda i: i['tool'] + i['description'])
 
+            found_details = list(filter(lambda i: isinstance(i['is_found'], dict), sorted_items))
+            all_pages = set()
+            for detail in found_details:
+                for set_items in detail['is_found'].values():
+                    all_pages.update(set_items)
+
             # Have issues
             issues_container = dom.find("div", {"id": "issues-container"})
             issues_banner = dom.find("div", {"id": "issues-banner"})
 
+            if found_details:
```

Comment:
Probably shouldn't be conditional. Leads to error if there are no found_details below:

2023-08-11 14:37:44,949 ERROR 789327 generate_conversion_report.py(261) local variable 'issues_detail_banner' referenced before assignment